lib/src/wizard/Wizard.tsx

```diff
@@ -329,7 +348,10 @@ const Description = styled.p`
   margin: 0;
 `;
 
-const StepSeparator = styled.div`
+type StepSeparatorProps = {
+  mode: Mode;
+};
+const StepSeparator = styled.div<StepSeparatorProps>`
```

Comment:
What do you think about removing the type and write directly `styled.div<{mode: Mode}>`?

Reply:
Done!